native/build.gradle

```diff
@@ -104,6 +125,7 @@ tasks.withType(Checkstyle) {
 
 spotbugsMain.finalizedBy validateSpotbugs
 checkstyleMain.dependsOn ':checkstyle:downloadCheckstyleRuleFiles'
+checkstyleTest.dependsOn ':checkstyle:downloadCheckstyleRuleFiles'
```

Comment:
Any need to duplicate this config?

Reply:
this is not a duplication. first one is for `checkstyleMain` and the other one is for `checkstyleTest`.